src/main/resources/travis/travis.sh

```diff
@@ -0,0 +1,17 @@
+#!/bin/bash
```

Comment:
Build binaries go under `src/bin`, not `src/main/resources`. The contents of the latter folder end up as embedded resources in the artifact jars.